contributor_environment.yml

```diff
@@ -4,26 +4,6 @@ channels:
   - defaults
 dependencies:
   - python=3.8
-  - osmnx
```

Comment:
I think we should keep these in the contributor conda file. The point of this is to serve as an alternative installation method if installation via pip doesn't work. Ideally we would only have a single installation method (like pip) but since some windows users have trouble installing via pip we should have this as an alternative.

Reply:
I'll back this change out and do some reading on `conda`.

Reply:
Reverted in https://github.com/NREL/mappymatch/pull/140/commits/8c1692e73dc64037ba37ec6994b8874c9a4c24e7